crates/turborepo-unescape/src/lib.rs

```diff
@@ -21,12 +24,19 @@ impl AsRef<str> for UnescapedString {
 }
 
 impl Deref for UnescapedString {
-    type Target = str;
+    type Target = String;
```

Comment:
I needed to access a method on `&mut String` so implementing `DerefMut` made sense.

`Deref` and `DeferMut` must have matching target types so in order to get a `&mut String` I needed `Deref` to target `String` instead of `str`. This is fine since `String` implements `Deref` with a `str` target so with deref coercion we can still go from `UnescapedString` to `str`.